functions-and-scope.js

```diff
@@ -14,9 +15,34 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe zorgt ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan bijhouden?
 // Log het antwoord in de terminal.
 
+function checkCumLaude (cijfers) {
+
+    let cumLaude = [];
+    let notCumLaude = [];
+
+for (let i = 0; i < cijfers.length ; i++) {
+
+    const cijfer = cijfers[i];
+
+        if( cijfer >= 8) {
+            cumLaude.push(cijfer);
+        } else {
+            notCumLaude.push(cijfer);
+        }
+
+    }
+
+return cumLaude.length;
+
+}
```

Comment:
Als we enkel geïnteresseerd zijn in de hoeveelheid en niet zozeer in de cijfers zelf, dan kunnen we net zo goed een counter bijhouden
```suggestion
function checkCumLaude (cijfers) {

    let counter = 0;

    for (let i = 0; i < cijfers.length ; i++) {

        const cijfer = cijfers[i];

        if( cijfer >= 8) {
            counter++;
        }
    }

    return counter;
}
```